ledger/error/generator/lib/src/test/scala/com/daml/error/generator/ErrorCodeDocumentationGeneratorSpec.scala

```diff
@@ -1,74 +0,0 @@
-// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package com.daml.error.generator
-
-import com.daml.error.utils.testpackage.subpackage.MildErrors
-import com.daml.error.utils.testpackage.subpackage.MildErrors.NotSoSeriousError
-import com.daml.error.utils.testpackage.{DeprecatedError, SeriousError}
-import com.daml.error.{Deprecation, Explanation, Grouping, Resolution}
-import org.scalatest.flatspec.AnyFlatSpec
-import org.scalatest.matchers.should.Matchers
-
-import scala.annotation.nowarn
-
-@nowarn("msg=deprecated")
-class ErrorCodeDocumentationGeneratorSpec extends AnyFlatSpec with Matchers {
```

Comment:
This class wasn't in any bazel target. 
Moved it to a different directory and fixed the test cases

Reply:
fyi @fabiotudone-da @tudor-da